### ISSUE TITLE
Some players lost inventories, XP, and Reputation after wipe.

### ISSUE THREAD

**Issue by Gsechs** [SmartlyDressedGames/Unturned-3.x-Community#763]
Looks like after the update, some (A shot in the dark - about 5-10%) of my players lost what they had in their inventory, all their skills, and their reputation.  

It's possible this may be because of a plugin I have running (AviSinglePlayerData), but I'm curious to know if this happened to anyone else.

Cheers.

**Comment by Ascivv1** [comment]
I don't think that is an issue that is directly affected by this update. Again, I don't think it is, but it might be.

**Comment by Gsechs** [comment]
> I don't think that is an issue that is directly affected by this update. Again, I don't think it is, but it might be.

Copy that. 

If anyone else had this problem, let me know please.
